_includes/neurips/2019/about.neurips.2019.html

```diff
@@ -40,7 +40,7 @@ <h2>Submission Instructions</h2>
 
 
         <h2>Submission Requirements</h2>
-        <p>All submissions will be double-blind (they should not contain author names) and may be up to two pages including all figures and tables, with an additional page for references. The papers must be submitted as a PDF created with the NeurIPS style format. All submissions must be in English. Submissions that do not follow these guidelines risk being rejected without consideration of their merits. Submissions must state the research problem, motivation, and technical contribution including experimental results when applicable. Submissions must be self-contained and include all figures, tables, and references. The submission deadline is 23:59 September 13th, 2019 AoE. Please note that no extensions will be offered for submissions.
+        <p>All submissions will be double-blind (they should not contain author names) and may be up to two pages including all figures and tables, with an additional page for references. The papers must be submitted as a PDF created with the <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles"> NeurIPS style format </a>. All submissions must be in English. Submissions that do not follow these guidelines risk being rejected without consideration of their merits. Submissions must state the research problem, motivation, and technical contribution including experimental results when applicable. Submissions must be self-contained and include all figures, tables, and references. The submission deadline is 23:59 September 13th, 2019 AoE. Please note that no extensions will be offered for submissions.
```

Comment:
The papers must be submitted as a PDF created by following the <a href="https://nips.cc/Conferences/2019/PaperInformation/StyleFiles">NeurIPS style guide</a> (template also available in <a href="https://www.overleaf.com/latex/templates/neurips-2019/tprktwxmqmgk">Overleaf</a>).